mne/preprocessing/infomax_.py

```diff
@@ -136,7 +148,8 @@ def infomax(data, weights=None, l_rate=None, block=None, w_change=1e-12,
     while step < max_iter:
 
         # shuffle data at each step
-        rng.seed(step)  # --> permutation is fixed but differs at each step
```

Comment:
I would just have gotten rid of this line.

to me you should set the init at the beginning if you want but that's it

@dengemann wdyt?